BuildHelpers/BuildHelpers.psm1

```diff
@@ -1,25 +1,37 @@
 #Get public and private function definition files.
-    $Public  = @( Get-ChildItem -Path $PSScriptRoot\Public\*.ps1 -ErrorAction SilentlyContinue )
-    $Private = @( Get-ChildItem -Path $PSScriptRoot\Private\*.ps1 -ErrorAction SilentlyContinue )
-    $ModuleRoot = $PSScriptRoot
+$Public  = @( Get-ChildItem -Path $PSScriptRoot\Public\*.ps1 -ErrorAction SilentlyContinue )
+$Private = @( Get-ChildItem -Path $PSScriptRoot\Private\*.ps1 -ErrorAction SilentlyContinue )
+$ModuleRoot = $PSScriptRoot
 
 #Dot source the files
-    Foreach($import in @($Public + $Private))
+Foreach($import in @($Public + $Private))
+{
+    Try
     {
-        Try
-        {
-            . $import.fullname
-        }
-        Catch
-        {
-            Write-Error -Message "Failed to import function $($import.fullname): $_"
-        }
+        . $import.fullname
     }
+    Catch
+    {
+        Write-Error -Message "Failed to import function $($import.fullname): $_"
+    }
+}
 
 # Load dependencies. TODO: Move to module dependency once the bug that
 # causes this is fixed: https://ci.appveyor.com/project/RamblingCookieMonster/buildhelpers/build/1.0.22
 # Thanks to Joel Bennett for this!
-    Import-Module $PSScriptRoot\Private\Modules\Configuration
+$fallbackModule = Get-Module -Name $PSScriptRoot\Private\Modules\Configuration -ListAvailable
+if ($configModule = Get-Module $fallbackModule.Name -ListAvailable)
+{
+    $configModule |
+        Where-Object { $_.Version -gt $fallbackModule.Version} |
+        Sort-Object -Property Version -Descending |
+        Select-Object -First 1 |
+        Import-Module -Force
+}
+if (-not (Get-Module $fallbackModule.Name | Where-Object { $_.Version -gt $fallbackModule.Version}))

```

Comment:
Hiyo!  Can we just do an else here?

Reply:
Nope!

Well.. yes, we can. But requires the `| Where Version -ge ...` to be in the if clause

Reply:
ohh!  I need coffee : )  works for me!